src/pages/home/home.page.tsx

```diff
@@ -1,21 +1,23 @@
 import React from 'react';
 import './home.styles.scss';
-
-import HomeCardHeader from '../../components/home-card-header/home-card-header.component';
 import HomeCardItems from '../../components/home-card-items/home-card-items.component';
 
-import card1Img from '../../assets/images/home-card-1.jpg';
-import card2Img from '../../assets/images/home-card-2.jpg';
-import card3Img from '../../assets/images/home-card-3.jpg';
+import { PeiliConsumer } from '../../context/context';
 
 const HomePage = () => {
   return (
-    <div className="homepage">
-      <HomeCardHeader />
-      <HomeCardItems image={card1Img} />
-      <HomeCardItems image={card2Img} />
-      <HomeCardItems image={card3Img} />
-    </div>
+    <PeiliConsumer>
+      {(value: any) => {
+        const { feedData, organizationsData } = value;
```

Comment:
There is an error in this line that causes the CI to fail. The `value` prop should be checked to ensure that isn't undefined. If it is, just render `null`.

Reply:
this was fixed